migration/sql-files/062-categories.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	name text NOT NULL
+);
+
+CREATE INDEX categories_id_index ON categories (id);
+CREATE UNIQUE INDEX categories_name_idx ON categories (name) WHERE deleted_at IS NULL;
+
+CREATE TABLE work_item_type_categories (
+	created_at timestamp with time zone,
+	updated_at timestamp with time zone,
+	deleted_at timestamp with time zone,
+	id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+	category_id uuid NOT NULL,
+	workitemtype_id uuid NOT NULL
+);
+
+CREATE INDEX work_item_type_categories_id_idx ON work_item_type_categories (id);
```

Comment:
@hectorj2f we already have one for categories_id -> https://github.com/almighty/almighty-core/pull/1037/files?diff=unified#diff-f39f9fd0e274c74d73da70aabf5dd364R9